### PR TITLE
Adds Additional Styling and Updates to README.md

### PR DESCRIPTION
**Pull request by nwthomas**
# Description

Updates Readme

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,107 +1,127 @@
...
+![MIT](https://img.shields.io/packagist/l/doctrine/orm.svg) ![React](https://img.shields.io/badge/react-v16.8.6-blue) [![Conflux](https://img.shields.io/badge/react--conflux-0.0.2--beta.6-blue)](https://www.npmjs.com/package/react-conflux) ![apollo-boost](https://img.shields.io/badge/apollo--boost-v0.4.3-blue) ![styled-components](https://img.shields.io/badge/styled--components-v4.3.2-ff69b4) [![grommet](https://img.shields.io/badge/design--system-v2.grommet.io-7D4CDB)](https://v2.grommet.io/) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier) ![eslint](https://img.shields.io/badge/eslint-airbnb-hotpink)
 
-🚫 The numbers 1️⃣ through 5️⃣ next to each item represent the week that part of the docs needs to be comepleted by.  Make sure to delete the numbers by the end of Labs.
+#
 
-🚫 Each student has a required minimum number of meaningful PRs each week per the rubric. Contributing to docs does NOT count as a PR to meet your weekly requirements.
+<h1 align="center"><img src="./assets/readme-header.png" /></h1>
```
Comment (cjbt) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -1,107 +1,127 @@
...
 
-# 1️⃣ Title of project goes here
+# Stampd Client
```
Comment (cjbt) [inline README.md]: What do you think about making the deployed site under this section? I feel like it's too low

File: client/src/components/Dashboard/SchoolDashboard/queries.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import client, { gql } from '../../../clientQuery';
+
```
Comment (cjbt) [inline client/src/components/Dashboard/SchoolDashboard/queries.js]: Hm I think this file is part of joes PR as well. I think we may need to Merge that before this and check whether it conflicts

**Comment by cjbt** [comment]
Hit that ready for review button if you're ready to merge the beauty

File: client/src/components/Dashboard/SchoolDashboard/queries.js (hunk 3 above)
Comment (nwthomas) [reply client/src/components/Dashboard/SchoolDashboard/queries.js]: Agreed - I'm behind master by 10 commits while ahead on README.md ones. I'll merge `master` with `readme` prior to the finishing the draft PR.

File: README.md (hunk 2 above)
Comment (nwthomas) [reply README.md]: That's a good idea - Updating now

Comment (nwthomas) [reply README.md]: Also, I had @byronholmes2018 put the `client` and `server` URLs in their respective repository headers:
image

Comment (cjbt) [reply README.md]: Ah I see it. I didn't notice it until now. Still, it could be an option. I don't tend to look at the headers usually in docs.

**Pull request merged**